angelsrefining/prototypes/recipes/refining-dynamic/pure-processing-mix.lua

```diff
@@ -80,7 +80,7 @@ data:extend(
       normal = {
         ingredients = {
           {type = "item", name = "angels-ore2-pure", amount = 2},
-          {type = "item", name = "angels-ore3-pure", amount = 2},
+          {type = "item", name = "angels-ore4-pure", amount = 2},
           {type = "item", name = "angels-ore5-pure", amount = 2},
           {type = "item", name = "catalysator-orange", amount = 1}
```

Comment:
I just noticed that the orange catalysator is unlocked with the Advanced ore refining 3 technology, which is inconsistent with the other catalysts.
![image](https://user-images.githubusercontent.com/26593477/122794394-7f846600-d2bc-11eb-87a5-cab63eac69f9.png)

I suggest unlocking it together with the geode processing 3 technology, utilizing the catallizer mk2, and add that technology as a prerequisite to advanced ore sorting 3.

Reply:
implemented in https://github.com/Arch666Angel/mods/pull/608/commits/017bc0f51e0ee12d5ae7f7d4fe8e9d9885cdd5b7
![image](https://user-images.githubusercontent.com/26593477/122813936-d9445a80-d2d3-11eb-90c3-049a3b946cc0.png)